lablqml/src_tests/test3.ml

```diff
@@ -0,0 +1,34 @@
+(* read test3.qml for explanation *)
+open OUnit
+open QmlContext
+
+let test3 () =
+  let (app,engine) = create_app_engine Sys.argv "src_tests/test3.qml" in
+  ignore app;
+  let _ =
+    let root = QQmlAppEngine.root_named engine in (* this function raises Failure if object is missing *)
+    let test = root "test" in
+    let nested = object_property_named test "nested" in
+    let nested_binding = Property.binding nested "nested_msg" (fun _ -> ()) in
+    print_endline ("nested message:" ^ (match Property.value nested_binding with `string s -> s | _ -> ""));
+    let mirror_root = root "mirror" in
```

Comment:
You use `mirror_root` only once. Maybe we can simplify the demo if we remove name conflict from QML file.